dir.props

```diff
@@ -45,19 +45,45 @@
     <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
```

Comment:
Steve and I are now on the same page about what these are intended for and I support only having the configurations that are needed for the given library.